frontend/src/App.js

```diff
@@ -226,6 +227,12 @@ export default function App() {
                   component={() => <Admin />}
                   role="Global Administrator"
                 />
+                <SecureRoute
```

Comment:
I think we dont want to acces this entrely on its own path but rather as a subPath on the admin page